agri-store-react/src/Components/ProductCard/ProductCard.css

```diff
@@ -0,0 +1,12 @@
+@import url("https://deploy-preview-13--agri-component.netlify.app/css/components.css");
```

Comment:
You don't need to import your component library in all CSS files. 
Importing on just App.css should be sufficient.